doc/extensions/acrbuildext/azext_acrbuildext/build.py

```diff
@@ -263,29 +268,46 @@ def acr_queue(cmd,
         result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
             build_request=build_request, resource_group_name=resource_group_name, registry_name=registry_name))
 
-        print("Queued a build with build-id: {}.".format(result.build_id))
+        size = os.path.getsize(tar_file_path)
+        unit = ""
+        for S in ['Bytes', 'KiB', 'MiB', 'GiB']:
+            if size < 1024:
+                unit = S
+                break
+            size = size / 1024.0
+        if unit == "":
+            unit = "GiB"
+
+        print("Sending build context ({0: .3f} {1}) to ACR Build as Id: {2}".format(
+            size, unit, result.build_id))
 
         if no_logs == False:
-            print("Starting to stream the logs...")
             return acr_build_show_logs(cmd, client, registry_name, result.build_id, resource_group_name)
     except Exception as err:
         raise CLIError(err)
```

Comment:
Extract the error message, this makes no difference from throwing the stack trace out.